modules/navier_stokes/include/base/NSFVBase.h

```diff
@@ -3484,9 +3485,6 @@ template <class BaseType>
 void
 NSFVBase<BaseType>::checkFrictionParameterErrors()
 {
-  if (parameters().isParamSetByUser("friction_types"))
-    checkDependentParameterError("friction_types", {"standard_friction_formulation"}, true);
```

Comment:
what's up with that? do we not want standard anymore?

Reply:
Standard is the default. We don't want to see an error when the user doesn't explicitly defines one in the action.

Reply:
ok sounds good